### PR TITLE
Cvsl 2207 afer address checks check your answers page

### PR DESCRIPTION
**Pull request by nishanthmahasamudram**

### CODE REVIEW

File: server/utils/nunjucksSetup.ts [hunk 1]
```diff
@@ -192,9 +192,5 @@ export function registerNunjucks(app?: express.Express): Environment {
     }
   })
```
Comment (nishanthmahasamudram) [inline server/utils/nunjucksSetup.ts:194]: removed unwanted to String functionality

File: integration_tests/checkYourAnswers.spec.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    await assessForEarlyRelease.stubGetAssessmentSummary(assessmentSummary(prisonNumber))
+    await assessForEarlyRelease.stubGetCheckRequestsForAssessment(prisonNumber, createCheckRequestsForAssessmentSummary)
+
+    await login(page, { authorities: ['ROLE_LICENCE_CA'] })
+
+    await page.goto(paths.prison.assessment.curfewAddress.checkYourAnswers({ prisonNumber }))
+
+    await expect(page).toHaveURL(
+      `${playwrightConfig.use.baseURL}/prison/assessment/A1234AE/curfew-address/check-your-answers`,
```
Comment (andrewrlee) [inline integration_tests/checkYourAnswers.spec.ts:24]: can we use `paths` for this?

File: integration_tests/checkYourAnswers.spec.ts [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+    await expect(page.getByRole('heading', { name: 'Preferred address' })).toBeVisible()
+    await expect(page.getByRole('heading', { name: 'Second address' })).not.toBeVisible()
+    await expect(page.getByTestId('prefered-address-residental-details')).toBeVisible()
+    await expect(page.getByText('second-address-residental-details')).not.toBeVisible()
+
+    const removeLink = await page.getByRole('link', { name: 'Remove preferred address' }).getAttribute('href')
+    expect(removeLink).toContain('/prison/assessment/A1234AE/curfew-address/1/check-your-answers/delete')
+
+    const chnageLink = await page.getByRole('link', { name: 'Change  change preferred' }).getAttribute('href')
```
Comment (andrewrlee) [inline integration_tests/checkYourAnswers.spec.ts:56]: and these two.. (also tiny typo)

File: server/views/pages/curfewAddress/checkYourAnswers.njk [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+{% from "partials/assessment/addressSummary.njk" import addressSummary %}
+{% from "partials/assessment/residentSummary.njk" import residentSummary %}
+{% from "govuk/components/summary-list/macro.njk" import govukSummaryList %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% set pageTitle = applicationName + " - Check your answers" %}
+{% set backLinkHref = "/" %}
+{% set addressCount = addressSummary.length %}
+{% set preferedAddress = checkRequestsForassessmentSummary | filter('preferencePriority', 'FIRST') %}
+{% set secondAddress = checkRequestsForassessmentSummary | filter('preferencePriority', 'SECOND') %}
```
Comment (andrewrlee) [inline server/views/pages/curfewAddress/checkYourAnswers.njk:12]: **suggestion:** you can `find` these so you don't have to deal with them as arrays

**Review by andrewrlee — APPROVED** [review]
LGTM

**Pull request merged**